### ISSUE TITLE
Shut-Down Turrets

### ISSUE THREAD

**Issue by Endermage77** [BEEmod/BEE2-items#1682]
This was mentioned by TeamSpen210 over in my suggestion on the Self-Fizzling Turret in #1612.

The Shut-Down turret will stop working whenever it recieves an input, and will start working when it does not. Or maybe the other way around. 
As an added bonus, why not let this still work when the turret deactivates by other means, such as being tipped?

**Comment by TeamSpen210** [comment]
I don't think you can turn on a turret after it's tipped over. Whenever it receives an input it'll turn on, unless inverted. I'll be implementing this on the normal turret item if possible.

**Comment by Byzarru** [comment]
For further reading, check out #1150.
